Sources/SparkConnect/SparkSession.swift

```diff
@@ -112,6 +112,26 @@ public actor SparkSession {
     return try await DataFrame(spark: self, sqlText: sqlText)
   }
 
+  /// Executes a SQL query substituting positional parameters by the given arguments, returning the
+  /// result as a `DataFrame`.
+  /// - Parameters:
+  ///   - sqlText: A SQL statement with positional parameters to execute.
+  ///   - args: An array of strings that can be converted to SQL literal expressions.
```

Comment:
`An array of strings`?

Do they must be strings? Looks like they can be any `Sendable`?

Reply:
Oh, right. It's typo.